docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
```

Comment:
I think we should template it using .env file

https://docs.docker.com/compose/environment-variables/#the-env-file

One again, feel fre

Reply:
I have added all parameters from .env to docker-compose.yml and tested with "vagrant up/destroy stage"